### PR TITLE
API-8220: document new system message

### PR DESCRIPTION
**Pull request by lpmichalik**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8220-customer-updated-sysmesg)
- [Jira](https://livechatinc.atlassian.net/browse/API-5117)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/messaging/references/system-messages/index.mdx [hunk 1]
```diff
@@ -79,6 +79,17 @@ The list of system messages sorted out alphabetically:
...
+| Customer’s name has been changed to %customer_name%                                          | Same as above but change made by integration                                                                       |
+| _%initiator% has changed customer's email to %customer_email%_                               | Customer's email was updated by agent when the chat was active with **Update Customer** request.                   |
+| Customer’s email has been changed to %customer_email%                                        | Same as above but change made by integration                                                                       |
+| _%initiator% has changed customer's name and email to %customers_name% and %customer_email%_ | Customer's name and email was updated by agent when the chat was active with a single **Update Customer** request. |
+| Customer’s name and email has been changed to %customer_email%                               | Same as above but change made by integration                                                                       |
```
Comment (janowsiany) [inline content/messaging/references/system-messages/index.mdx:91]:
```suggestion
| Customer’s name and email has been changed to %customers_name% and %customer_email%                               | Same as above but change made by integration                                                                       |
```
right?

**Review by janowsiany — CHANGES_REQUESTED** [review]

File: content/messaging/references/system-messages/index.mdx (hunk 1 above)
Comment (lpmichalik) [reply content/messaging/references/system-messages/index.mdx]: good catch, but no ;)  good catch though

**Review by ale-kacperczyk — APPROVED** [review]

**Review by janowsiany — APPROVED** [review]
I think that italics are broken but I think it applies to the whole section cc: @ale-kacperczyk

**Comment by ale-kacperczyk** [comment]
> I think that italics are broken but I think it applies to the whole section cc: @ale-kacperczyk

@janowsiany  Italics seem to be working fine. Do you mean `%` sign? It's a placeholder for the value. Maybe it's not very clear what it means

**Pull request merged**